src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+  padding: 14px;
+  padding-top: 20px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  perspective: 160px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 9px auto 45px;
+    transition: transform 400ms;
+
+    &:hover {
+      transform: translate3d(0,10px,32px);
+      border: 3px solid #f3f3f3;
+    }
+  }

```

Comment:
add transparent border and box-sizing to make transition more smoother

https://www.loom.com/share/d23d25545d7a4261a789d5fdec3f44f1